src/components/ListGroup.js

```diff
@@ -94,14 +101,14 @@ export default function ListGroup(props) {
 
       let newOrdering = ordering;
 
-      // open if not open and configured
-      if (!open && shouldOpen) {
+      // open if not open, configured and triggered by add button
+      if (addTriggered && !open && shouldOpen) {
```

Comment:
Feels really like an awkward hack. Let's document it like that and add a `// TODO(marstamm): ...`.

Reply:
I added a `todo` and a bit of documentation to move the logic out of the ListGroup

Reply:
Thanks @marstamm.

With these comments (and this commit as a reference) the changes look manageable. 

Let's try to keep our hacks per LOC as low as possible.

[![image](https://user-images.githubusercontent.com/58601/227270853-a7cee52d-8ad0-4973-acf5-19058242b5de.png)](https://xkcd.com/1513/)